hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -378,9 +388,12 @@ public String toString() {
                 + ", getCount=" + getCount
                 + ", putCount=" + putCount
                 + ", removeCount=" + removeCount
-                + ", totalGetLatencies=" + totalGetLatencies
-                + ", totalPutLatencies=" + totalPutLatencies
-                + ", totalRemoveLatencies=" + totalRemoveLatencies
+                + ", totalGetLatencies=" + totalGetLatencies + " microseconds"
```

Comment:
I someone relies on this for automatic parsing, it will break anyway, when the unit is changed (silently).